main.js

```diff
@@ -0,0 +1,84 @@
+var board = {
+  pegs: [[5, 4, 3, 2, 1],
+  [],
+  []],
+
+  moveDisc: function(startPeg, endPeg) {
+  board.pegs.map(peg => {
+    var pegLine = "---";
+    var newArr = [pegLine].concat(peg);
+    return newArr;
+  })
```

Comment:
@ninavtan - I'm not sure what the point of doing this is. `map` returns a new array, which you're not capturing in a variable or elsewhere.